bentoml/frameworks/tensorflow.py

```diff
@@ -246,23 +240,39 @@ def pack(
                         "version 1",
                         str(options),
                     )
-
                 tf.saved_model.save(
                     obj, self._tmpdir.name, signatures=signatures,
                 )
             self._path = self._tmpdir.name
         else:
             self._path = obj
-
+        self._packed = True
+        loaded = self.get()
+        logger.warning(
+            f"Due to TensorFlow's save and load mechanism, only methods marked by "
+            f"`tf.function` (and some keras defaults) could be restored after "
```

Comment:
"Due to TensorFlow's internal mechanism, only methods annotated with `@tf.function` and Keras model API methods including `predict` and `xxxx` can be restored after a save & load."